test/model/message_list_test.dart

```diff
@@ -0,0 +1,210 @@
+import 'package:checks/checks.dart';
+import 'package:test/scaffolding.dart';
+import 'package:zulip/api/model/events.dart';
+import 'package:zulip/api/model/model.dart';
+import 'package:zulip/api/route/messages.dart';
+import 'package:zulip/model/message_list.dart';
+import 'package:zulip/model/narrow.dart';
+import 'package:zulip/model/store.dart';
+import '../api/fake_api.dart';
+import '../api/model/model_checks.dart';
+import '../model/binding.dart';
+import '../model/test_store.dart';
+import '../example_data.dart' as eg;
+
+const int userId = 1;
+const int streamId = 2;
+
+Future<PerAccountStore> setupStore(ZulipStream stream) async {
+  addTearDown(TestZulipBinding.instance.reset);
```

Comment:
Adding the teardown so it lives in the same location as the setup, for clarity

Reply:
Sounds good!